packages/atlas-react/src/common/styles/_colors.scss

```diff
@@ -2,14 +2,10 @@
 
 $colors: map-get($tokens, 'color');
 
-:root {
```

Comment:
all of this is now built ad `atlas-tokens` level, which makes more sense... right?